src/components/PivotTable/Aggregators/Aggregators.tsx

```diff
@@ -0,0 +1,93 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+
+import { Radio } from '../../Radio'
+import { Select } from '../../Select'
+import { VFlow } from '../../VFlow'
+import { useLocale } from '../../../i18n'
+import { Aggregator, KeyMap } from './model-aggregator'
+import { getAggregators, getKeyDependentAndNotDependentAggregators } from './util-aggregator'
+
+type AggregatorsProps<T extends object> = {
```

Comment:
```suggestion
interface AggregatorsProps<T extends object> = {
```
We usually prefer using interfaces for component props

Reply:
Could you verify if this is still displayed in Storybook without the `export`?